client/my-sites/sidebar/sidebar.jsx

```diff
@@ -326,7 +326,15 @@ module.exports = React.createClass( {
 			linkClass += ' is-paid-plan';
 		}
 
-		let planName = site.plan.product_name_short;
+		let planName = site.plan.product_name_short,
+			labelClass;
+
+		if ( abtest( 'plansUpgradeButton' ) === 'button' && productsValues.isFreePlan( site.plan ) ) {
+			labelClass = 'add-new';
+			planName = 'Upgrade'; // TODO: translate this string if the test is removed
+		} else {
+			labelClass = 'plan-name';
+		}
 
 		if ( productsValues.isFreeTrial( site.plan ) ) {
 			planName = this.translate( 'Trial' );
```

Comment:
I noted that on my todo list but since you're updating this file could you add more context to this translation? I guess just explaining it's for free trials for plans would really help translators to deliver a more accurate translation.